src/Repository/IssueRepository.php

```diff
@@ -41,11 +46,16 @@ public function findAll()
     {
         $result = $this->client->get('issues.json?include=children,attachments,relations,changesets,watchers');
 
-        if (! $result) {
+        if(! $result) {
```

Comment:
We ignore the PSR-2 in every if statement at this point. Maybe we should reformat the code in another branch.